common/lib/xmodule/xmodule/lti_module.py

```diff
@@ -338,14 +368,49 @@ class LTIModule(LTIFields, LTI20ModuleMixin, XModule):
 
         Otherwise error message from LTI provider is generated.
     """
+    resources_dir = None
 
-    js = {
+    preview_view_js = {
         'js': [
             resource_string(__name__, 'js/src/lti/lti.js')
-        ]
+        ],
+        'xmodule_js': resource_string(__name__, 'js/src/xmodule.js'),
+    }
+    preview_view_css = {
+        'scss': [
+            resource_string(__name__, 'css/lti/lti.scss')
+        ],
     }
-    css = {'scss': [resource_string(__name__, 'css/lti/lti.scss')]}
-    js_module_name = 'LTI'
+
+    mako_template = 'widgets/metadata-only-edit.html'
+
+    studio_js_module_name = 'MetadataOnlyEditingDescriptor'
+    studio_view_js = {
+        'js': [
+            resource_string(__name__, 'js/src/raw/edit/metadata-only.js')
+        ],
+        'xmodule_js': resource_string(__name__, 'js/src/xmodule.js'),
+    }
```

Comment:
Using the JS from [MetadataOnlyEditingDescriptor](https://github.com/edx/edx-platform/blob/046feb0cf0c02acb562739bb9128755755ef3157/common/lib/xmodule/xmodule/editing_module.py#L113) class.